app/views/org_admin/sections/_index.html.erb

```diff
@@ -38,7 +33,7 @@
               <%# This is AJAX loaded on demand unless section_id is specified in the URL %>
               <% if current_section.present? && section.id == current_section.id %>
                 <% partial = "org_admin/sections/#{section.modifiable? ? 'edit' : 'show'}" %>
-                <%= render partial: partial, 
+                <%= render partial: "#{partial}", 
```

Comment:
If we are now always displaying 'show' do we need the variable partial here? If I remember well I added this logic to display the active section (if any passed through URL)...